src/test/java/uk/gov/digital/ho/hocs/info/api/TopicResourceTest.java

```diff
@@ -112,6 +113,10 @@ public void getTopicByUUID() {
         }};
     }
 
+    private List<ParentTopicAndTopicDto> getParentTopicDtos() {
+        return getParentTopics().stream().map(pt -> ParentTopicAndTopicDto.from(pt)).collect(Collectors.toList());
+    }
```

Comment:
Can this method be removed?

Reply:
Didn't notice this, yeah it looks like this should be removed.

Reply:
resolved this in a new pr